qtodotxt/lib/tasklib.py

```diff
@@ -44,6 +44,11 @@ def _reset(self):
         self.due_error = ""
         self.threshold = None
         self.keywords = {}
+        self.recMode = None
+        self.recModeCompl = 0   # Original due date mode: Task recurs from original due date
+        self.recModeOrDue = 1   # Completion date mode: Task recurs from completion date
+        self.recIncr = None
+        self.recInterv = None
```

Comment:
Maybe you should have 2 class called Recurtion with 3 members: Mode (an Enum), Increment and Interval (Do not use so much shortcuts, it make it very hard to understand)
The would recMode be None if no recursion and a Recurtion instance if task is recuring. There are other possibilities but we definitely do not need 5 intance variables